src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -79,6 +81,31 @@ const DetailsModalStackNavigator = () => (
     </DetailsModalStack.Navigator>
 );
 
+const ParticipantsModalStackNavigator = () => (
+    <ParticipantsModalStack.Navigator
+        path={ROUTES.PARTICIPANTS}
+    >
+        <ParticipantsModalStack.Screen
+            name="Participants_Root"
+            component={ParticipantsPage}
+            options={{
+                ...defaultSubRouteOptions,
```

Comment:
We should be able to pass these to the navigator instead and prevent having to add this stuff to every screen.